DESCRIPTION

```diff
@@ -39,9 +39,7 @@ Imports:
     pbapply,
     zip,
     jsonlite,
-    parallel,
-    lifecycle,
-    stats
+    lifecycle
```

Comment:
Any particular reason to remove `parallel`?

Reply:
Both stats and parallel are base packages and these packages doesn't have to be imported.